### ISSUE TITLE
Privacy and accessibility policy can require impossible contract agreements

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#697]
### What are you trying to do that you can't?

There are privacy and accessibility laws that result in school districts requiring platforms to comply with complex contracts that guarantee particular things about the platform. For example, some contracts say things like "you promise to pay our lawyer fees if we are sued for a violation of this privacy or accessibility policy" or "you promise that if we find an accessibility problem you will fix it immediately." These requirements are not feasible to sign legally, because they make unkeepable promises, since this is not a for-profit project and it has no revenue stream.

### What is your idea?

I have no idea what to do about this. Neither did anyone at the SIGCSE birds of a feather session. One possibility was writing defensively on our about page, or providing templated agreements around specific policies, to help give districts and school leaders confidence about our promises. We could try writing some of these around privacy and accessibility law.

### Design

See [approved design](https://github.com/wordplaydev/wordplay/issues/697#issuecomment-2968171714).

**Comment by sdahl4** [comment]
@amyjko Hello! Unfortunately due to my new work schedule I can't make Wednesday meetings this quarter but I would still like to be assigned to research this if possible? I would love to schedule a zoom to discuss this issue.

**Comment by amyjko** [comment]
Assigned! Researching this would be very helpful. I'm short on meeting time — the world is in chaos right now, and it's demanding a lot of conflict management — but DM me on LCI Slack and we can see if we can find a time for a short chat.

**Comment by sdahl4** [comment]
This is a problem without a perfect solution, with current resources for this project.

 I did quite a bit of research looking for rope holes for non profits with no revenue but there isn’t anything I found for this specific situation.

A ‘bandaid’ solution for right now like you suggested, is too right defensively about our program in our about the resources and ability we have to comply with certain policies. 

We could add a section like this to the website. 

On School District Requirements and Regulations

Wordplay is a nonprofit organization with no revenue stream. Because of that there are certain agreements that we cannot legally comply with, like paying lawyer fees is a school is sued for a policy violation, or fixing issues on a certain schedule. 

While we understand school districts have their own specific regulations for accessibility and privacy, as a whole we are constantly working to make sure all of our features are ADA compliant, and students information is only collected optionally, and that data is only used for account access. 

Here is our response to common school regulation policies 

*templates here*

What common regulation policies do you think it would be a good idea to provide some templated agreements for?

**Comment by sdahl4** [comment]
I think that sounds very reasonable. What I suggest is a mailto link to a template email that a school IT can fill out and send. Since to my knowledge wordplay does not have an email, would it make sense for them to go to your UW email? 

This way we don't have to build a whole system within Wordplay to receive these issue requests.  Would you believe this to be a common enough issue to run to that it would make sense for us to  build that type of system?

This is what I suggest to add to our about page:

On School District Requirements and Regulations

Wordplay is a nonprofit organization with no revenue stream. Because of that there are certain agreements that we cannot legally comply with, like paying lawyer fees is a school is sued for a policy violation, or fixing issues on a certain schedule. 

While we understand school districts have their own specific regulations for accessibility and privacy, as a whole we are constantly working to make sure all of our features are ADA compliant, and students information is only collected optionally, and that data is only used for account access. 

We acknowledge that this platform is constantly updating and shifting as we improve and add features.  If there is an issue regarding a school district requirement or certain regulation you would like to reach out to us about please fill out the template email in the link below. 

<img src="https://github.com/user-attachments/assets/7c395588-0148-4de7-9594-e2067afa2142" width="150" />


The link would than take them to this email template: 

Subject: Wordplay Issue Inquiry 

Please fill out the needed areas of this form: 

School or District Name:

Your Name:

Your Role: 

Is there a regulation or school requirement we need to meet that we currently do not?

What requirement/regulation do do we need to meet that we currently can’t?

Is there any other information that you would like to give us about this issue?

**Comment by amyjko** [comment]
I like this basic design, for several reasons:

* It keeps humans in the loop
* It makes a clear point of entry for IT staff to contact us
* It provides information that school IT needs to make decisions
* It gives them a route to provide feedback that's more direct than the creator feedback mechanism

We'd probably have to continually tweak the language on the site as we learn, so that's not something we have to finalize. And I think your interpretation is right: we can't feasibly meet the contract agreements, so a design that just sets boundaries on what we can do is probably the next best thing.

Approving this design.

**Comment by amyjko** [comment]
Marking buildable and opening for assignment.
